src/features/content-explorer/content-explorer-modal-container/ContentExplorerModalContainer.js

```diff
@@ -54,6 +54,8 @@ class ContentExplorerModalContainer extends Component {
         onSelectItem: PropTypes.func,
         /** Folder is in the process of being created. */
         isCreatingFolder: PropTypes.bool,
+        /** Whether the user can see the Include Subfolders toggle with updated design */
+        isIncludeSubfoldersAllowed: PropTypes.bool,
```

Comment:
how about something like `canIncludeSubfolders` as the prop name? present tense verbs with an `is` prefix sound a little awkward to me